### ISSUE TITLE
"Cannot find module 'eslint-plugin-react'" when trying to run standard

### ISSUE THREAD

**Issue by BinaryBen** [standard/standard#1329]
**What version of standard?**

Not sure, I assume latest. Below error appears event when trying to run what I pressume should be a version command (`standard -v` or `standard -V`)

**What operating system, Node.js, and npm version?**

macOS Sierra 10.12.6, v11.1.0, 6.9.0 respectively

**What did you expect to happen?**

I assume a list of things that didn't meet standard's standard

**What actually happened?**

```
standard: Unexpected linter output:

Error: Failed to load plugin 'react' declared in '--config » eslint-config-standard-jsx': Cannot find module 'eslint-plugin-react'
Require stack:
{ ... }
If you think this is a bug in `standard`, open an issue: https://github.com/standard/standard/issues
```

Let me know if you need the full output within { ... }

I have already tried installing `eslint` and `eslint-plugin-react` globally but that did nothing.

**Comment by feross** [comment]
Thanks for the issue. There is a bug right now where `standard` doesn't work when installed globally. Looking into it now.

**Comment by feross** [comment]
This should be fixed in `standard` 13.0.2.

**Comment by felipezacani** [comment]
Just bumped into this issue here:
    "eslint": "^6.7.2",
    "eslint-config-prettier": "^4.3.0",
    "eslint-config-standard": "^14.1.0",
    "eslint-plugin-import": "^2.19.1",
    "eslint-plugin-node": "^10.0.0",
    "eslint-plugin-prettier": "^3.1.1",
    "eslint-plugin-promise": "^4.2.1",
    "eslint-plugin-react": "^7.17.0",
    "eslint-plugin-standard": "^4.0.1"
